lib/view/edit_item_dialog.dart

```diff
@@ -20,6 +20,7 @@ class _EditItemDialogState extends State<EditItemDialog> {
   @override
   void initState() {
     _editingController = TextEditingController(text: widget.item.value);
+    _editingController.addListener(() => setState(() {}));
```

Comment:
What is the purpose of this?

Reply:
This is so that when the user either types something into the text field when it was previously blank or erases what was typed (so that the field becomes blank), the submit button either becomes enabled (in the first case) or disabled (in the second). 

Should I add a comment?

Reply:
Alright! Yes, please add some kind of comment like `Update the submit button state when the user input changes`